src/components/Signup/Signup.test.js

```diff
@@ -35,5 +35,69 @@ describe('Signup', () => {
 
     // Assert
     expect(container.querySelectorAll('.error')).not.toHaveLength(0);
+  });  
+
+  it('should show Argentina below Argelia', () => {
```

Comment:
What do you think about 

```suggestion
  it('should show Argentina below Argelia when selected language is es', () => {
```